test/runnable/auto1.d

```diff
@@ -1,4 +1,11 @@
 /*
+TEST_OUTPUT:
+---
+runnable/auto1.d(26): Deprecation: `scope` as a type constraint is deprecated.  Use `scope` at the usage site.
+runnable/auto1.d(75): Deprecation: `scope` as a type constraint is deprecated.  Use `scope` at the usage site.
+runnable/auto1.d(106): Deprecation: `scope` as a type constraint is deprecated.  Use `scope` at the usage site.
+runnable/auto1.d(110): Deprecation: `scope` as a type constraint is deprecated.  Use `scope` at the usage site.
+---
```

Comment:
Maybe remove the `scope` on the declarations instead? This tests seems to target stack allocated class instances (`scope x = new ...`), not `scope class`.